src/sparseml/recipe_template/utils.py

```diff
@@ -12,20 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any, Dict, Optional
```

Comment:
I'd love to see some unit testing for this batch of four PRs. Both for hardening and documentation purposes.